data_curation/README.md

```diff
@@ -1,4 +1,350 @@
-# DIBBS Text to Code data_curation
+# DIBBS Text to Code (TTC) - Data Curation
 
-The `data_curation` folder contains scripts for TTC model development, tuning, and
-evaluation.
+## Table of Contents
+
+- [Overview](#overview)
+- [Scripts](#scripts)
+- [Data Files](#data-files)
+   - [LOINC](#loinc)
+   - [LOINC Part Synonyms & Abbreviations](#loinc-part-synonyms-&-abbreviations)
+   - [LOINC Part Descriptions](#loinc-part-descriptions)
+   - [LOINC UMLS Related Names](#loinc-umls-related-names)
+   - [SNOMED](#snomed)
+   - [HL7](#hl7)
+- [Instructions](#instructions)
+   - [Generating SNOINC Extracts](#generating-snoinc-extracts)
+     - [Dependencies](#dependencies)
+     - [Command Line](#command-line)
+     - [Direct Relma DB Queries](#direct-relma-db-queries)
+
+## Overview
+
+The `data_curation` folder contains scripts for TTC model development, tuning, and evaluation.  Most of the scripts leverage data that is being pulled from the LOINC, UMLS, and HL7 APIs.  However, some require the LOINC RelmaDB (MS-Access database).
+
+## Scripts
+
+### terminology_valueset_sync.py
+
+Contains various functions to pull data from SNOMED, LOINC, and HL7 APIs to provide data for the TTC model. For detailed instructions on how to use the various scripts [see instruction section below](#command-line)
+
+### augmentation.py
+
+A gathering of various functions to modify data from the different terminology data sets to help with model training and tuning.  Functions that randomly scramble words, randomly delete characters, and randomly replacing words with related terms are included.  The `configs.py` is leveraged to allow the user to configure the various parameters/properties for these random data modification functions.
+
+### configs.py
+
+A collection of various configurations used to augment data and/or create synthetic data, using the terminology data sets.
+
```

Comment:
Is there any more detail we should add here?

Reply:
I bet @bamader will have more to say here!